float32 widaq0
float32 widaq1
float32 widaq2
float32 widaq3
float32 positionx
float32 positiony
float32 yaw
